pkg/kapp/cmd/tools/inspect_tree_view.go

```diff
@@ -133,7 +134,7 @@ func newAssocSortingValue(resource ctlres.Resource, rsByUID map[string]ctlres.Re
 }
 
 func (a *assocSortingValue) Value() string {
-	return strings.Join(a.Values(), "@")
+	return a.Values()[0] + "-" + fmt.Sprintf("%d", a.Depth())
```

Comment:
Should we check the length of the array before using an index?

Reply:
It does make sense to. I think this might be an issue if a resource that should be displayed does not have a `kapp.k14s.io/association` label. Looking into this and a better approach as this one seems to miss the case where there are multiple resources at the same depth.